src/spaceone/identity/conf/global_conf.py

```diff
@@ -24,6 +24,7 @@
 
 IDENTITY = {
     'token': {
+        'temporary_token_timeout': 3600,
```

Comment:
change key of token configurations
`temporary_token_timeout` -> `verify_code_timeout`